### DISCUSSION TITLE
Android 12 workaround

### DISCUSSION THREAD

**Discussion by ceaserone** [termux/termux-app#3132] [General]
Not only am I getting the known issues on Android 12 I am also having crash on update issues...I had gotten a workaround with 12 barely and updating packages is closing termux entirely again....damb this 12 any help!? If not I'm going to actually buy a new phone.

**Comment by sylirre** [comment]
The only issue related to Android 12 which affects Termux is [phantom process killer](https://github.com/termux/termux-app/issues/2366). See the linked ticket for overview and solutions.
